core/shared/src/main/scala/scalaz/zio/Queue.scala

```diff
@@ -382,6 +394,8 @@ object Queue2 {
         )
         .uninterruptible
 
+    final val isShutdown: UIO[Boolean] = checkShutdownState.map(_ => false) <> UIO.succeed(true)
```

Comment:
You can simplify this: `shutdownHook.map(_.isDefined)`

Reply:
thx, done